extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeMixPlaylistExtractorTest.java

```diff
@@ -101,14 +111,14 @@ public void getContinuations() throws Exception {
             InfoItemsPage<StreamInfoItem> streams = extractor.getInitialPage();
             final Set<String> urls = new HashSet<>();
 
-            //Should work infinitely, but for testing purposes only 3 times
+            // Should work infinitely, but for testing purposes only 3 times
             for (int i = 0; i < 3; i++) {
                 assertTrue(streams.hasNextPage());
                 assertFalse(streams.getItems().isEmpty());
 
                 for (final StreamInfoItem item : streams.getItems()) {
-//                    TODO Duplicates are appearing
-//                    assertFalse(urls.contains(item.getUrl()));
+                    // TODO Duplicates are appearing
```

Comment:
Shouldn't this be addressed?

Reply:
I don't know, maybe YouTube suggestions return sometimes the same stream even if there is the VISITOR cookie.

Reply:
Sorry, this is for another PR